code/modules/events/immovable_rod.dm

```diff
@@ -259,24 +252,6 @@ In my current plan for it, 'solid' will be defined as anything with density == 1
 
 	if(iscarbon(smeared_mob))
 		var/mob/living/carbon/smeared_carbon = smeared_mob
-		//SKYRAT ADDITION - RODS DEAL LESS DAMAGE TO MOBS LAYING DOWN
-		var/first_time_around = TRUE
-		if(smeared_carbon.client)
-			var/client/cli = smeared_carbon.client
-			if(cli in already_hit)
-				first_time_around = FALSE
-			else
-				already_hit |= cli
-		if(smeared_carbon.resting && first_time_around)
-			var/cur_damage = smeared_carbon.getBruteLoss() + smeared_carbon.getFireLoss()
-			var/damage = (smeared_carbon.maxHealth * maxhp_damage_resting) - cur_damage
-			if(damage > 0)
-				smeared_carbon.adjustBruteLoss(damage)
-			// This copies functionality from below, essentially a 10% chance you just get fucked big time
-			if(smeared_mob.density || prob(10))
-				smeared_mob.ex_act(EXPLODE_HEAVY)
-			return
-		//SKYRAT ADDITION - END
```

Comment:
While I disagree with using a list of clients rather than just a list of mobs (because clients can be deleted and stuff), I think this was done at Xyel's request? I'm not 100% sure though.

Reply:
It needs to not be shitcode.